lib/devise-security/models/expirable.rb

```diff
@@ -22,7 +22,11 @@ module Expirable
 
       # Updates +last_activity_at+, called from a Warden::Manager.after_set_user hook.
       def update_last_activity!
-        self.update_column(:last_activity_at, Time.now.utc)
+        if defined? ActiveRecord
```

Comment:
IMO a better check would be to see if `update_column` is defined.

Reply:
Do you prefer having a check like this?

I have to leave the elsif check because both active_record and mongoid have update_attribute method.

```
      # Updates +last_activity_at+, called from a Warden::Manager.after_set_user hook.
      def update_last_activity!
        if respond_to?(:update_column)
          self.update_column(:last_activity_at, Time.now.utc)
        elsif defined? Mongoid
          self.update_attribute(:last_activity_at, Time.now.utc)
        end
      end
```

Reply:
done